recipes/systemd_service.rb

```diff
@@ -69,5 +69,6 @@ class ::Chef::Recipe
       'AccuracySec' => "#{node['chef_client']['splay']}sec",
     }
   )
-  action(timer ? [:create, :enable, :start] : [:disable, :delete])
+  action(timer ? [:create, :enable, :start] : [:stop, :disable, :delete])
+  notifies :restart, to_s, :delayed
```

Comment:
this looks like it's missing something with the to_s ? maybe I'm missing something here.